### PR TITLE
Chores(locales): Update En UI

### PR DESCRIPTION
**Pull request by Sherlock113**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

En UI update

### CODE REVIEW

**Comment by Sherlock113** [comment]
/assign @leoendless
Thank you.

File: src/components/Cards/Containers/Item.jsx [hunk 1]
```diff
@@ -210,7 +210,7 @@ export default class ContainerItem extends React.Component {
           <div>
             {isUndefined(detail.restartCount) ? '-' : detail.restartCount}
           </div>
-          <p>{t('Restart Count')}</p>
+          <p>{t('Restart Times')}</p>
```
Comment (leoendless) [inline src/components/Cards/Containers/Item.jsx:213]: Restart Count is used by k8s offical tool kubectl. No need change the word.

Comment (Sherlock113) [reply src/components/Cards/Containers/Item.jsx:213]: ok, got it

**Pull request merged**